Assets/MXR.SDK/Editor/Scene Export/SceneExporter.cs

```diff
@@ -47,22 +56,29 @@ public static void ExportScene(string scenePath, string outputFilePath, BuildTar
                 Directory.Delete(exportDir, recursive: true);
             Directory.CreateDirectory(exportDir);
 
+            // Build and export the asset bundles to the export directory
             var manifest = BuildPipeline.BuildAssetBundles(
                 exportDir,
                 new AssetBundleBuild[] { sceneBuild, assetsBuild },
                 BuildAssetBundleOptions.None,
                 buildTarget
             );
+
+
             if (manifest == null) {
-                Debug.LogError("Asset bundle build failed");
-            } else {
-                ZipUtils.CompressDirectory(exportDir, outputFilePath);
-                if (deleteExportDir)
-                    Directory.Delete(exportDir, recursive: true);
-                else
-                    Debug.Log("Exported intermediate asset bundles and manifests to " + exportDir);
-                Debug.Log("Exported .mxrus to " + outputFilePath);
+                Debug.unityLogger.Log(LogType.Error, TAG, "Asset bundle build failed");
```

Comment:
Could we make this? 

Debug.unityLogger.Log(LogType.Error, TAG, "Asset bundle build failed, manifest is null?  ");

Reply:
Added a build report summary in #123 that could be more descriptive